lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -721,6 +745,10 @@ def draw_rubberband(self, x0, y0, x1, y1):
         NavigationToolbar2GTK3.draw_rubberband(
             self._make_classic_style_pseudo_toolbar(), None, x0, y0, x1, y1)
 
+    def remove_rubberband(self, x0, y0, x1, y1):
```

Comment:
I'm confused by the signature on this method.

Reply:
Ah, oops, copy&paste error.